src/Container/Aggregate/AggregateRepositoryFactory.php

```diff
@@ -94,27 +86,27 @@ public function __invoke(ContainerInterface $container)
         $aggregateType = AggregateType::fromAggregateRootClass($config['aggregate_type']);
         $aggregateTranslator = $container->get($config['aggregate_translator']);
 
-        $snapshotStore = isset($config['snapshot_store'])? $container->get($config['snapshot_store']) : null;
+        $snapshotStore = isset($config['snapshot_store']) ? $container->get($config['snapshot_store']) : null;
 
-        $streamName = isset($config['stream_name'])? new StreamName($config['stream_name']) : null;
+        $streamName = isset($config['stream_name']) ? new StreamName($config['stream_name']) : null;
 
-        $oneStreamPerAggregate = isset($config['one_stream_per_aggregate'])? (bool)$config['one_stream_per_aggregate'] : false;
+        $oneStreamPerAggregate = (bool) ($config['one_stream_per_aggregate'] ?? false);
 
         return new $repositoryClass($eventStore, $aggregateType, $aggregateTranslator, $snapshotStore, $streamName, $oneStreamPerAggregate);
     }
 
     /**
      * @inheritdoc
      */
-    public function dimensions()
+    public function dimensions(): array
```

Comment:
interop-config interfaces are defined without return type hints. Maybe we should omit they here.